.github/workflows/release_rh_community_bundle.yml

```diff
@@ -0,0 +1,83 @@
+name: Red Hat Community bundle
+on:
+  workflow_call:
+    secrets:
+      RH_COMMUNITY_OPERATORS_TOKEN:
+        required: true
+    inputs:
+      version:
+        description: "The version to release, without the leading `v`"
+        required: true
+        type: string
+      previous_version:
+        description: "The previous version, used for the CVS's `replaces` field, without the leading `v`"
+        required: true
+        type: string
+      skip_range_lower:
+        description: "Lower bound for the skipRange field in the CSV, should be set to the oldest supported version, without the leading `v`"
+        required: true
+        type: string
+
+permissions:
+  contents: write
+
+jobs:
+  make_rh_community_bundle:
+    name: Build and commit PR for bundle in Red Hat Community
+    runs-on: ubuntu-22.04
+    env:
+      PROJECT: ${{ github.repository }}
+      VERSION: ${{ inputs.version }}
+      PREVIOUS_VERSION: ${{ inputs.previous_version }}
+      SKIP_RANGE_LOWER: ${{ inputs.skip_range_lower }}
+    steps:
+      - name: Log inputs
+        run: |
+          echo "Building version: ${VERSION}"
+          echo "which replaces version: ${PREVIOUS_VERSION}"
+          echo "Lower skip range bound: ${SKIP_RANGE_LOWER}"
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          path: operator
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: operator/go.mod
+
+      - name: Checkout Community operators
```

Comment:
```suggestion
      - name: Checkout Red Hat Community Operators
```

Reply:
We might want to use this kind of automation for OperatorHub community, so it would be better to different between the Red Hat community and the Kubernetes community